meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
You should generally use css instead of ` <br></br>`. Also, you can just do `<br/>`